frontend/dockerfile/dockerfile2llb/convert.go

```diff
@@ -1381,6 +1391,8 @@ func dispatchCopy(d *dispatchState, cfg copyConfig) error {
 
 	var a *llb.FileAction
 
+	validateCopySourcePath(cfg.opt.dockerIgnorePatterns, cfg.params.SourcePaths, cfg.location, lint)
```

Comment:
This is only for copies from build context, not multi-stage copies for example (that should be covered by tests).

Reply:
Do you mind pointing me to where multi-stage copies are being handled so I can get that covered? :O